pkg/logs/launchers/integration/launcher.go

```diff
@@ -29,15 +32,28 @@ var endOfLine = []byte{'\n'}
 // Launcher checks for launcher integrations, creates files for integrations to
 // write logs to, then creates file sources for the file launcher to tail
 type Launcher struct {
-	sources              *sources.LogSources
-	addedConfigs         chan integrations.IntegrationConfig
-	stop                 chan struct{}
-	runPath              string
-	integrationsLogsChan chan integrations.IntegrationLog
-	integrationToFile    map[string]string
+	sources                   *sources.LogSources
+	addedConfigs              chan integrations.IntegrationConfig
+	stop                      chan struct{}
+	runPath                   string
+	integrationsLogsChan      chan integrations.IntegrationLog
+	integrationToFile         map[string]*FileInfo
+	fileSizeMax               int64
+	combinedUsageMax          int64
+	combinedUsageSize         int64
+	leastRecentlyModifiedTime time.Time
```

Comment:
Do we need `leastRecentlyModifiedTime` if `leastRecentlyModifiedFile` stores `lastModified` already? 
I actually don't think this property is used anywhere other than being updated in `updateLeastRecentlyModifiedFile`

Reply:
You're right it was unnecessary, deleted 👍